### DISCUSSION TITLE
Allow example.com emails in mock data

### DISCUSSION THREAD

**Discussion by digitarald** [community/community#50673] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Seems like Copilot stops at emails, but in this case I am creating mock data and `@example.com` is a valid email for testing purposes:

![image](https://user-images.githubusercontent.com/8599/226702372-bd01b2e4-4aa4-49bf-acf2-9aecfb98b413.png)

**Comment by andrewpmk** [comment]
I've noticed this bug as well. Is it intentional that Copilot refuses to generate email addresses or a bug? I frequently create mock data for unit tests containing emails and this is annoying.

Example:
Copilot refuses to generate a fake email for Sally Smith.

```
contact_info = [
    {
        "first_name": "John",
        "last_name": "Doe",
        "email": "johndoe@example.com",
        "phone": "555-555-5555",
        "address": "123 Main St",
        "city": "Anytown",
        "state": "TX",
        "zip_code": "77777",
    },
    {
        "first_name": "Jane",
        "last_name": "Doe",
        "email": "janedoe@example.com",
        "phone": "444-444-4444",
        "address": "321 Main St",
        "city": "Othertown",
        "state": "TX",
        "zip_code": "88888",
    },
    {
        "first_name": "Sally",
        "last_name": "Smith",
        "email": "
```

**Comment by chscott** [comment]
Same here, but we use `example.org`.
